src/generated/v3/jobService/jobServiceFetcher.ts

```diff
@@ -0,0 +1,96 @@
+export type JobServiceFetcherExtraProps = {
```

Comment:
See the note in the README about this. This file should be mostly empty. Take a look at userServiceFetch.ts for an example.

Reply:
readed and follow the instructions for this.